lib/faker.rb

```diff
@@ -13,17 +13,31 @@
 
 module Faker
   module Config
+    SAFE_DOMAIN_SUFFIX = 'example'
+
     @locale = nil
     @random = nil
+    @internet_safe_mode = false
 
     class << self
       attr_writer :locale, :random
+      # force domain suffix to be ".example" to prevent potential security issue.
+      # As Faker is extensively used in automated test environment, we don't want to use real email address (potentially) for testing
+      #
+      # @example:
+      #   Faker::Config.internet_safe_mode = true
+      #   Faker::Config.internet_safe_mode? # => true
```

Comment:
```suggestion
      # @example:
      #   Faker::Config.internet_safe_mode = true
```

I don't see the benefits of having both. What do you think?